packages/service/tests/unit/test_discovery_client.py

```diff
@@ -395,6 +397,67 @@ def test___no_registered_measurements___enumerate_services___returns_empty_list(
     assert not available_measurements
 
 
+@pytest.mark.parametrize("programming_language", ["Python", "LabVIEW"])
+def test___service_registered___resolve_service_with_information___sends_request(
```

Comment:
Since this test accepts programming language as a parameter. Should the given phrase be, "registered_measurements" instead of "service_registered"?

Reply:
Sure, Updated.